pkg/validate/raw/validate/namespace_validator.go

```diff
@@ -27,7 +27,7 @@ import (
 // to the following rules:
 // - if the object is a Namespace, it must not be `config-management-system`
 // - if the object has a metadata namespace, it must be a valid k8s namespace
-//   and it must not be in `config-management-system`
+// and it must not be in `config-management-system`
```

Comment:
This should be updated to reflect that "we allow other declared resources in the `config-mangemente-system` namespace`.

Reply:
the comment that indicates the restriction has been removed